auxiliary_files/run_scripts/mumble-overlay.in

```diff
@@ -1,6 +1,6 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Environment poisoning is a thing. Please leave it as `/bin/bash`.

Reply:
> Environment poisoning is a thing

What's that? Never heard of it before :eyes:

Reply:
If the user calls /usr/bin/env to derive where bash is, it might get a user-overridden bash (e.g. a custom script in PATH called bash that executes some malicious code).